pkg/hubble/observer/local_observer.go

```diff
@@ -44,26 +45,6 @@ import (
 // in their init() function.
 var DefaultOptions []observeroption.Option
 
-// GRPCServer defines the interface for Hubble gRPC server, extending the
-// auto-generated ObserverServer interface from the protobuf definition.
-type GRPCServer interface {
-	observerpb.ObserverServer
-	// Start starts the server and blocks.
-	Start()
-	// GetEventsChannel returns the channel to push monitor events to.
-	GetEventsChannel() chan *flowpb.Payload
-	// SetEventsChannel sets the events channel. For unit testing only.
-	SetEventsChannel(chan *flowpb.Payload)
-	///GetRingBuffer returns the underlying ring buffer to parsed events.
-	GetRingBuffer() *container.Ring
-	// GetStopped returns a channel that gets closed at the end of the
-	// main server loop after all the events have been processes. Used
-	// in unit testing.
-	GetStopped() chan struct{}
-	// GetLogger returns the logger assigned to this gRPC server.
-	GetLogger() logrus.FieldLogger
-}
-
```

Comment:
I like this diff :)
However, several methods comments of `LocalObserverServer` [1] now have incorrect comments such as "Start implements GRPCServer.Start".

[1]: This struct should really be renamed at this point but this is outside the scope of this PR.

Reply:
Yeah let's update the comments and naming in a separate PR.